### ISSUE TITLE
​Self-Hosted: Failed to pull docker image

### ISSUE THREAD

**Issue by tiagodj** [toeverything/AFFiNE#6841]
### What happened?

When following the instructions [here](https://github.com/toeverything/docker/blob/master/README.md), I get the error below:

```
➜  ~ docker pull ghcr.io/toeverything/affine-self-hosted:latest
Error response from daemon: Head "https://ghcr.io/v2/toeverything/affine-self-hosted/manifests/latest": unauthorized
```

Is this image not publicly accessible?

Someone else opened [this](https://github.com/toeverything/docker/issues/1) but looks like no one is checking that repo.

Thanks

### Distribution version

macOS ARM 64 (Apple Silicon)

### Are you self-hosting?

- [X] Yes

### Relevant log output

```shell
Error response from daemon: Head "https://ghcr.io/v2/toeverything/affine-self-hosted/manifests/latest": unauthorized
```

**Comment by nekomiya-kasane** [comment]
I also encounter this problem.

**Comment by miladhatami1393** [comment]
I also encounter this problem.

**Comment by miladhatami1393** [comment]
I also encounter this problem

**Comment by gongxiansen** [comment]
Is it solved？

**Comment by Brooooooklyn** [comment]
`master` branch is staled, see https://docs.affine.pro/docs/self-host-affine for the latest doc

**Comment by InfamousUser** [comment]
> see https://docs.affine.pro/docs/self-host-affine for the latest doc

These instructions are lacking.

**Comment by caijunqian** [comment]
I encountered this problem again.
```
-> # docker pull ghcr.io/toeverything/affine-self-hosted:latest
Error response from daemon: Head "https://ghcr.io/v2/toeverything/affine-self-hosted/manifests/latest": unauthorized
```

**Comment by Azradex** [comment]
Docs are broken please fix. Same.

Update:

Found a working image from the Coolify template. 

ghcr.io/toeverything/affine-graphql:stable

**Comment by smallyaohailu** [comment]
Error response from daemon: Head "https://ghcr.io/v2/toeverything/affine-self-hosted/manifests/latest": unauthorized

**Comment by HikiJun97** [comment]
Same either.
`Error response from daemon: Head "https://ghcr.io/v2/toeverything/affine-graphql/manifests/stable": denied: denied`

**Comment by Glitter230207** [comment]
1. Is it solved?
2. I have this 403 Forbidden issue.

**Comment by IDESTIINY** [comment]
> 1. Is it solved?
> 
>     2. I have this 403 Forbidden issue.

@Glitter230207 
I had the issue with affine hosted on my coolify instance.
I fixed it by changing the docker image as described in the docs - https://docs.affine.pro/self-host-affine/install/upgrade
Quote:

> Starting from version 0.23.0, AFFiNE docker image switched from affine-graphql to affine. Please manually modify it in your compose file or download the latest one from [GitHub release](https://github.com/toeverything/affine/releases/latest/download/docker-compose.yml).

Apparently I had version <23 before - I wasn't aware.
This fixed the issue for me

**Comment by thumpnail** [comment]
Late to the discussion, might be useful if someone still looking for a solution...
Anyway
I wanted to run affine on docker on my Ugreen Nas and had the denied: denied error.
BUT i could pull the image on my windows machine so i pulled it there, then used docker save to save it. them moved it to the docker compose project folder on my Nas and then used docker load to load the generated image tar file. and then i deployed without pulling the latest images.
Tldr.:
On windows: 
``docker pull ghcr.io/toeverything/affine:stable``
``docker save ghcr.io/toeverything/affine:stable -o affine-stable.tar``
-> Moving the tar to the nas
-> ``ssh`` into the nas and ``cd`` to the folder where the tar archive lies
On ssh/Nas:
``sudo docker load -i ./affine-stable.tar``
and then i think ``docker compose up`` but i did it through the ugreen docker app.
